dev/entrypoint.sh

```diff
@@ -9,4 +9,6 @@ printf "\nMigrations:\n"
 python openslides_backend/migrations/migrate.py finalize
 printf "\n"
 
+global/meta/dev/scripts/apply_db_schema.sh
+
```

Comment:
This one we don't want. Sometimes it would leave your database corrupt, sometimes not. 
Finally all changes on an existing database have to be made with migrations, never with applying structural changes without data migration.
But we should talk about this one, soon.

Reply:
This is a remainder from a previous version of my PR. If you rebase this PR onto the current `feature/relational-db` branch, this should be gone

Reply:
Done